spec/forms/placement_form_spec.rb

```diff
@@ -16,12 +21,122 @@
 
     context "when there are two existing placements" do
       before do
-        allow(trainee).to receive(:placements).and_return(build_list(:placement, 2))
+        allow(placements_form).to receive(:placements).and_return(build_list(:placement, 2))
       end
```

Comment:
At the top where the rest of the lets are
```
  let(:placements) { [] }
  let(:trainee) { create(:trainee, placements:) }
```

Then replace the before block 
```
let(:placements) { build_list(:placement, 2) }
```

or just this 

```
let(:trainee) { create(:trainee, placements: build_list(:placement, 2)) }
```